Gopkg.toml

```diff
@@ -1,31 +1,36 @@
+
 [[constraint]]
-  name = "github.com/bitrise-io/go-utils"
   branch = "master"
+  name = "github.com/bitrise-io/go-utils"
 
 [[constraint]]
-  name = "github.com/bitrise-io/goinp"
   branch = "master"
+  name = "github.com/bitrise-io/goinp"
 
 [[constraint]]
-  name = "github.com/bitrise-tools/go-xamarin"
   branch = "master"
+  name = "github.com/bitrise-tools/go-xamarin"
 
 [[constraint]]
+  branch = "master"
   name = "github.com/bitrise-tools/go-xcode"
-  branch = "xcode-managed-codesign-group"
 
 [[constraint]]
-  name = "github.com/pkg/errors"
   branch = "master"
+  name = "github.com/bitrise-tools/xcode-project"
+
+[[constraint]]
+  name = "github.com/pkg/errors"
+  version = "0.8.1"
 
 [[constraint]]
   name = "github.com/spf13/cobra"
-  branch = "master"
+  version = "0.0.3"
```

Comment:
please lock for master branches and run `dep ensure -update`, until we do not come up with a new agreement